modele-social/règles/salarié.yaml

```
@@ -159,43 +161,152 @@ contrat salarié . frais professionnels . titres-restaurant . contrôle taux emp
   formule: taux participation employeur > 60%
   description: La part employeur du titre-restaurant doit être de 60% au maximum
 
-contrat salarié . frais professionnels . indemnité kilométrique vélo:
-  icônes: 🚴
-  question: Le salarié profite-t-il de l'indemnité kilométrique vélo pour se rendre au travail ?
+contrat salarié . frais professionnels . abonnement transports publics:
+  icônes: 🚍
+  valeur: oui
+
+contrat salarié . frais professionnels . abonnement transports publics . montant:
+  titre: Abonnement aux transports publics
+  question: Quel montant le salarié dépense-t-il en abonnement aux transports publics chaque mois ?
+  unité: €/mois
+  par défaut: 0 €/mois
   description: |
-    Cette indemnité n'est pour l'instant pas obligatoire.
+    L'employeur doit prendre en charge 50% du montant dépensé par le salarié pour les transports publics lui permettant de se rendre sur son lieu de travail.
+    
+    Cette prise en charge (dans la limite des 50% du montant) est exonérée de cotisations sociales et d'impôt sur le revenu.
 
-    L'employeur a le choix de la mettre en place dans son entreprise.
+    Dans le cas d'un temps partiel, le taux de prise en charge sera le même pour un mi-temps ou plus. En dessous, le taux de prise en charge sera proportionnel.
+  références:
+    Articles R3261-1 à -10 du code du travail, version 01/01/2009: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000020080272/2009-01-01
+    Article 81 du code des impôts, version en vigueur au 31/12/2020: https://www.legifrance.gouv.fr/codes/id/LEGIARTI000042910732/2020-12-31/
+
+contrat salarié . frais professionnels . abonnement transports publics . taux de prise en charge:
+  titre: Taux de prise en charge
+  valeur:
+    produit:
+      assiette: 
+        le minimum de:
+          - temps de travail . quotité de travail
+          - 50%
+      taux: 2 * 50%
+
+contrat salarié . frais professionnels . abonnement transports publics . prise en charge:
+  titre: Abonnement transports publics, part prise en charge par l'employeur (déductible)
+  unité: €/mois
+  valeur: taux de prise en charge * montant
 
-    Pour bénéficier de l'indemnité de 200€ retenue dans ce calcul, le salarié doit faire 4km (aller-retour) de vélo pour se rendre au travail chaque jour travaillé.
+contrat salarié . frais professionnels . transports personnels:
+  valeur: oui
 
-    Cette indemnité est cumulable avec le remboursement des frais de transport en commun s'il s'agit d'un trajet à vélo de rabattement vers une station de transport.
+contrat salarié . frais professionnels . transports personnels . montant:
+  valeur:
```

```suggestion
  titre: Transports personnels
  valeur:
```

(sinon la règle apparait uniquement sous le titre "montant" dans la doc